processor/transformer/transformer.go

```diff
@@ -140,6 +144,7 @@ func (trans *HandleT) Setup() {
 	trans.logger = pkgLogger
 	trans.sentStat = stats.DefaultStats.NewStat("processor.transformer_sent", stats.CountType)
 	trans.receivedStat = stats.DefaultStats.NewStat("processor.transformer_received", stats.CountType)
+	trans.cpDownGauge = stats.DefaultStats.NewStat("processor.control_plane_down_gauge", stats.GaugeType)
```

Comment:
Please prepare the relevant operator changes for the 2 new alerts that we are introducing